dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -8,10 +8,10 @@ def perform_evaluation
 
   def submit_evaluation
     authorize! :read, Plc::Course
```

Comment:
We've discussed this in a previous PR, but `:read` probably isn't the right permission to check here.

I know permissions are still WIP, but may be worth adding a TODO so it make more sense in code reviews (and we don't forget to update it later).